telegram/_bot.py

```diff
@@ -2758,6 +2759,14 @@ async def edit_message_live_location(
                 if specified.
             reply_markup (:class:`telegram.InlineKeyboardMarkup`, optional): An object for a new
                 inline keyboard.
+            live_period (:obj:`int`, optional): New period in seconds during which the location
+                can be updated, starting from the message send date. If `0x7FFFFFFF` is specified,
```

Comment:
```suggestion
                can be updated, starting from the message send date. If `0x7FFFFFFF` is specified,
```

We'll need a new constant for this value

Reply:
Done.